src/core/components/spinner/spinner.tsx

```diff
@@ -21,6 +21,11 @@ export interface ISpinnerProps extends HTMLAttributes<HTMLDivElement> {
      * @default neutral
      */
     variant?: SpinnerVariant;
+    /**
+     * Defines if the spinner is in the loading state or not.
+     * @default true
+     */
+    isLoading?: boolean;
```

Comment:
What is the usecase for a non-loading spinner? Pedantic, but if the usease is much less common, could make a case for using that as the prop and setting it to false by default. It's a bit strange seeing isLoading on the Spinner lol